recipes/podofo/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "0.9.6":
+    url: "https://netcologne.dl.sourceforge.net/project/podofo/podofo/0.9.6/podofo-0.9.6.tar.gz"
```

```suggestion
    url: "https://sourceforge.net/projects/podofo/files/podofo/0.9.6/podofo-0.9.6.tar.gz/download"
```
I did not test it, but it should work because this kind of link is used in https://github.com/conan-io/conan-center-index/blob/master/recipes/freetype/all/conandata.yml#L5 for example.
It has the advantage that you do not depend on a specific mirror

---

this writes file as `download` not tar.gz
```
-rw-rw-r--  1 marcin marcin 973K lip  8  2018 download
-rw-rw-r--  1 marcin marcin 973K lip  8  2018 podofo-0.9.6.tar.gz

```